apps/dashboard/app/models/batch_connect/app.rb

```diff
@@ -152,13 +142,17 @@ def delete_cached_cluster?(cached_cluster)
         (self.clusters.size == 1 && self.clusters[0] != cached_cluster)
     end
 
-
     # The clusters that the batch connect app can use. It's a combination
     # of what the app is configured to use and what the user is allowed
-    # to use.
-    # @return [OodCore::Clusters] clusters available to the app user
+    # to use. If the app has clusters configured, it preserves the order
+    # when it can. glob configurations' order may be platform dependent.
+    #
+    # @return [Array<OodCore::Cluster>] clusters available to the app user
```

Comment:
Looks like the interface changes here - but not actually. The previous implementation returns the same, so it's actually the comment that's out of date.

```ruby
OodAppkit.clusters.select { |cluster| cluster_allowed(cluster) }
```